src/react/databrowser/objects/details/Properties.jsx

```diff
@@ -17,6 +19,10 @@ const TruncatedValue = styled(T.Value)`
 `;
 
 function Properties({ objectMetadata }: Props) {
+  const dispatch = useDispatch();
+  const prefixWithSlash = usePrefixWithSlash();
+  const isLegalHoldEnabled = objectMetadata.legalHoldStatus === 'ON';
```

Comment:
I'm wondering wether we should not keep this ON logic on the client side and everywhere else use a boolean.
In src/js/S3Client.js we could change :
```
getObjectLegalHold(bucketName, objectKey, versionId) {
    const params = {
      Bucket: bucketName,
      Key: objectKey,
      VersionId: versionId,
    };

    return new Promise((resolve, reject) => {
      this.client.getObjectLegalHold(params, (error, data) => {
        if (error) {
          if (
            error.code === 'NoSuchObjectLockConfiguration' ||
            error.code === 'InvalidRequest'
          ) {
            return resolve('OFF');
          }
          return reject(error);
        }
        if (data.Status) {
          return resolve(data.Status);
        }
        return resolve('OFF');
      });
    });
  }
```

To 
```
getObjectLegalHold(bucketName, objectKey, versionId) {
    const params = {
      Bucket: bucketName,
      Key: objectKey,
      VersionId: versionId,
    };

    return new Promise((resolve, reject) => {
      this.client.getObjectLegalHold(params, (error, data) => {
        if (error) {
          if (
            error.code === 'NoSuchObjectLockConfiguration' ||
            error.code === 'InvalidRequest'
          ) {
            return resolve(false);
          }
          return reject(error);
        }
        if (data.Status) {
          return resolve(data.Status === 'ON');
        }
        return resolve(false);
      });
    });
  }
```

Then everywhere else we wouldn't have to take care of the boolean translation of the value

Reply:
Yeah. It makes sense to store the boolean `isLegalHoldEnabled`